ci-operator/step-registry/ccm/conf/apply-feature-gate/ccm-conf-apply-feature-gate-commands.sh

```diff
@@ -23,8 +76,41 @@ spec:
   customNoUpgrade:
     enabled:
     - ExternalCloudProvider
+    - CSIMigrationAWS
```

Comment:
Does this mean we can only run this on AWS? Can we add the other CSIMigration<platform> feature gates so we could test this on other platforms too?

Reply:
Added OpenStack to the list. These featureGates should not overlap, as the cluster is usually one platform of the two.